src/app/shared/http-services/group-permissions.service.ts

```diff
@@ -8,7 +8,7 @@ import * as D from 'io-ts/Decoder';
 import { pipe } from 'fp-ts/function';
 import { decodeSnakeCase } from 'src/app/shared/operators/decode';
 
-const permissionsDecoder = pipe(
+export const permissionsDecoder = pipe(
```

Comment:
I think this permission decoder should be in a helper so that both services can import it without having a dependency of one service on the other

Reply:
Fixed